.github/workflows/workflow.yml

```diff
@@ -1,11 +1,6 @@
 name: AEA framework sanity checks and tests
 
-on:
-  push:
-    branches:
-      - develop
-      - main
-  pull_request:
+on: pull_request
```

Comment:
Changed to all PRs. 

No point in running checks and tests on pushes because a) we almost always make changes via PRs, b) the workflow doesn't make sense (we don't check CI after a push).